src/firefly/js/visualize/ui/ColorTableDropDownView.jsx

```diff
@@ -221,40 +282,62 @@ const AdvancedColorPanel= ({allowPopout}) => {
             setContrast(newContrast);
         }
 
-        setColorTableId(Number(colorTableId));
+        setColorTableId(colorTableId);
         setUseRed(useRed);
         setUseGreen(useGreen);
         setUseBlue(useBlue);
         const colorChangeParam=  {
-                        plotId:plot.plotId,
-                        cbarId: Number(colorTableId),
-                        bias: band===Band.NO_BAND ? newBias : newBiasAry,
-                        contrast: band===Band.NO_BAND ? newContrast : newContrastAry,
-                        useRed, useBlue, useGreen,
-                    };
+            plotId:plot.plotId,
+            cbarId: colorTableId,
+            bias: band===Band.NO_BAND ? newBias : newBiasAry,
+            contrast: band===Band.NO_BAND ? newContrast : newContrastAry,
+            nanPixelColor,
+            useRed, useBlue, useGreen,
+        };
         image ? dispatchColorChange(colorChangeParam) : dispatchColorChangeThrottled(colorChangeParam);
     };
 
     const ctArray= (image? colorTables : hipsColorTables);
 
-    const makeItems = () =>
-        ctArray.map((ct) =>
-            (<ToolbarButton icon={ct.icon ? <img src={ct.icon} style={{height:8, width:'18.5rem'}}/> : undefined}
-                            tip={ct.tip} style={{padding: '2px 0 2px 0'}}
-                            text={ct.icon ? undefined : 'Default Color Map'}
-                            enabled={true} horizontal={false} key={ct.id}
-                            hasCheckBox={true} checkBoxOn={colorTableId === ct.id}
-                            onClick={() => changeBiasContrastColor(ct.id, bias, contrast)}/>));
+    const makeItems = () => {
+        return ctArray.map((ct) => {
+            if (ct.ignore) return undefined;
+            return <ColorItem {...{key:ct.id,colorTableId:colorTableId+'',ct,bias,contrast,changeBiasContrastColor}}/>;
+        }).filter(Boolean);
+    };
 
-    const makeAdvancedStandardFeatures= () => (
+    const r= isReversedColor(colorTableId+'');
+    const allColorTableIDs=[];
+    ctArray.forEach(({id,disableReverse,ignore}) => {
+        if (ignore) return;
+        allColorTableIDs.push(id);
+        if (disableReverse) return;
+        allColorTableIDs.push(id+REVERSED_END_CHAR);
+    });
+    const cSlideVal= allColorTableIDs.findIndex( (v) => v===colorTableId);
+    const cBarTitle= (ctArray.find( (ct) => ct.id===colorTableId || ct.id+REVERSED_END_CHAR===colorTableId)?.tip ?? '') + (r ? '- reversed' : '');
```

Comment:
```suggestion
    const cBarTitle= (ctArray.find( (ct) => ct.id===colorTableId || ct.id+REVERSED_END_CHAR===colorTableId)?.tip ?? '') + (r ? ' - reversed' : '');
```
Add space before `-` too

Reply:
thanks for spotting that